src/main/kotlin/org/elm/lang/core/types/TypeInference.kt

```diff
@@ -288,12 +304,22 @@ private class InferenceScope(
             }
 
     private fun inferFunctionCall(expr: ElmFunctionCallExpr): Ty {
+        if (expr.hasErrors) return TyUnknown()
+
         val targetTy = inferAtom(expr.target)
         val arguments = expr.arguments.toList()
 
         // always infer the arguments so that they're added to expressionTypes
         val argTys = arguments.map { inferAtom(it) }
 
+        if (targetTy is TyVar) {
```

Comment:
When does this happen?

Reply:
When calling an unannotated parameter e.g. `foo f = f 123`